### DISCUSSION TITLE
Bank Tags - Add Jump to Top/Bottom Buttons

### DISCUSSION THREAD

**Discussion by Xylot** [runelite/runelite#17296] [Ideas]
It would be awesome to have a jump to top and bottom button for bank tags. I have a ton and navigating through them would be a bit cleaner if I could jump around to the end anchors as opposed to going to the macro view or scrolling around.<img width="68" alt="Screenshot 2023-12-20 224937" src="https://github.com/runelite/runelite/assets/14049503/bd10832d-f725-48f7-8cfc-89bb44cf530d">

**Comment by umer-rs** [comment]
Adding a feature to make scrolling quicker may also help. Move up/down with each scroll scrolling the entire group of 5, or however many are there.
